crates/winscard/src/scard_context.rs

```diff
@@ -377,7 +314,7 @@ impl<'a> ScardContext<'a> {
                 (compressed.len() + 2 /* unknown flags */ + 2/* uncompressed certificate len */) as u32;
             value.extend_from_slice(&total_value_len.to_le_bytes());
 
-            value.extend_from_slice(&[0x01, 0x00]); // unknown flags
+            value.extend_from_slice(&[0x01, 0x00]); // flags that specify that the certificate is compressed
```

Comment:
question: Do you have a document to link referencing this?

Reply:
Nope, I don't have. It's just my experience. During the investigation, I noticed that the WinSCard returns an uncompressed cert on Windows for the Yubikey smart card. The only difference in cache headers was this flag